### ISSUE TITLE
If I click two links hit back can't get back to first page

### ISSUE THREAD

**Issue by ianorlin** [agateau/mup#7]
To create a restrutured text file with a linkable url in it somewhere else that itself has a link on it then click that second link then then right click back with the context menu. You will be able to click forward but not one to get back to original page.

**Comment by agateau** [comment]
I am not sure I get it, can you maybe rephrase it? Also, where did you find a context menu with a "back button"?
